bakery/README.md

```diff
@@ -29,13 +30,13 @@ and
 will provide information about the command, defaults, and its positional and nonpositional arguments.
 
 ### Generate a pipeline file for a particular environment
-Run `./build pipeline <env> [options]...`
+Run `./build pdf-pipeline <env> [options]...`
 
 The choices for `<env>` are the basenames of the `.json` files in the `env/` directory.
 
 Examples:
-- `./build pipeline prod` -> Build the pipeline with prod environment variables and output on stdout.
-- `./build pipeline staging -o pipeline.staging.yml` -> Build the pipeline with staging environment variables and output to file `pipeline.staging.yml`, overwriting the file if it exists.
+- `./build pdf-pipeline prod` -> Build the pipeline with prod environment variables and output on stdout.
+- `./build pdf-pipeline staging -o pipeline.staging.yml` -> Build the pipeline with staging environment variables and output to file `pipeline.staging.yml`, overwriting the file if it exists.
```

Comment:
Super minor, but should we make this `-o pdf-pipeline.staging.yml` to align with the renaming of those files?